demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -221,6 +226,17 @@ def save(self):
     def ignored_errors(self) -> list:
         raise NotImplementedError
 
+    def ignored_errors_related_files(self, file_path: str) -> list:
```

Comment:
```suggestion
    def ignored_errors_related_files(self, file_path: Union[str, Path]) -> list:
```

Path should be allowed too i suppose